pkg/controller/deployment/sync.go

```diff
@@ -109,7 +109,7 @@ func (dc *DeploymentController) getAllReplicaSetsAndSyncRevision(deployment *ext
 	// List the deployment's RSes & Pods and apply pod-template-hash info to deployment's adopted RSes/Pods
 	rsList, podList, err := dc.rsAndPodsWithHashKeySynced(deployment)
 	if err != nil {
-		return nil, nil, fmt.Errorf("error labeling replica sets and pods with pod-template-hash: %v", err)
+		return nil, nil, fmt.Errorf("error labeling ReplicaSets and pods with pod-template-hash: %v", err)
```

Comment:
In this sentence, given that we use "pods", using "replica sets" is more consistent.